src/packages/admin-ui-components/package.json

```
@@ -53,20 +57,33 @@
 		"react-dom": "^18.0.0 || ^19.0.0"
 	},
 	"devDependencies": {
+		"@storybook/addon-essentials": "^8.6.12",
+		"@storybook/addon-interactions": "^8.6.12",
+		"@storybook/addon-onboarding": "^8.6.12",
+		"@storybook/blocks": "^8.6.12",
+		"@storybook/manager-api": "8.6.12",
+		"@storybook/react": "^8.6.12",
+		"@storybook/react-vite": "^8.6.12",
+		"@storybook/test": "^8.6.12",
```

Let's go ahead and pin these dependencies to specific versions so they don't drift on us, e.g.
```suggestion
		"@storybook/test": "8.6.12",
```